src/seedu/addressbook/ui/TextUi.java

```diff
@@ -92,28 +85,28 @@ public String getUserCommand() {
     public void showWelcomeMessage(String version, String storageFilePath) {
         String storageFileInfo = String.format(MESSAGE_USING_STORAGE_FILE, storageFilePath);
         showToUser(
-                DIVIDER,
-                DIVIDER,
+                format.getDivider(),
+                format.getDivider(),
                 MESSAGE_WELCOME,
                 version,
                 MESSAGE_PROGRAM_LAUNCH_ARGS_USAGE,
                 storageFileInfo,
-                DIVIDER);
+                format.getDivider());
     }
 
```

Comment:
TextUI is essentially still doing the job of formatting. You're mostly just placing the constants within Formatter class, which is more like encapsulation rather than following SRP.